xinference/types.py

```diff
@@ -17,6 +17,15 @@
 from typing_extensions import Literal, NotRequired, TypedDict
 
 
+class Image(TypedDict):
+    url: Optional[str]
+    b64_json: Optional[str]
+
+
+class ImageList(TypedDict):
+    data: List[Image]
```

Comment:
Besides `data`, there should be another field `created`.

Here's an example from OpenAI's doc:
```json
{
  "created": 1589478378,
  "data": [
    {
      "url": "https://..."
    },
    {
      "url": "https://..."
    }
  ]
}
```

Reply:
Fixed. Thanks.